guides/common/modules/proc_upgrading-postgresql-on-the-external-database.adoc

```diff
@@ -0,0 +1,14 @@
+[id="Upgrading_PostgreSQL_on_the_External_Database_{context}"]
+= Upgrading PostgreSQL on the external database
+
+. If you use an external database, upgrade the external database to PostgreSQL 13.
+For more information, see {RHELDocsBaseURL}8/html/deploying_different_types_of_servers/using-databases#fast-upgrade-using-the-pg_upgrade-tool_migrating-to-a-rhel-8-version-of-postgresql[Fast upgrade using the pg_upgrade utility] in _Deploying different types of servers_.
```

Comment:
This procedure module needs an introduction and the `.Procedure` heading. Both are mandatory elements of procedure modules.

Now I'm thinking about how to do this without too much duplication. How about this:

```suggestion
= Upgrading PostgreSQL on the external database

If you use an external database, upgrade the external database to PostgreSQL 13.

.Procedure
. Follow the instructions in {RHELDocsBaseURL}8/html/deploying_different_types_of_servers/using-databases#fast-upgrade-using-the-pg_upgrade-tool_migrating-to-a-rhel-8-version-of-postgresql[Fast upgrade using the pg_upgrade utility] in _{RHEL}{nbsp}8 Deploying different types of servers_ to upgrade the external database.
```

Reply:
I used `If you use an external database, upgrade the external database and rectify the permissions.` 
WDYT?